src/vs/workbench/browser/actions/workspaceActions.ts

```diff
@@ -131,6 +131,29 @@ export class GlobalRemoveRootFolderAction extends Action {
 	}
 }
 
+
+export class ShowFileInFolderAction extends Action {
```

Comment:
in general we try to minimize change to VS Code source code under `vs`  directory to make refreshing their source code easier.  Could you please move these under `sql` somewhere?